spec/rubocop/rspec/example_spec.rb

```diff
@@ -17,6 +17,16 @@ def example(source)
       .to eq(s(:str, 'does x'))
   end
 
+  it 'extracts interpolated doc string' do
```

Comment:
I've rarely seen it in the wild, but sometimes a Symbol is passed as the first argument, and RSpec considers it a docstring, not metadata, at least as far as I remember.
It feels that that node pattern change covers `sym` pretty much, not sure if an additional example would prevent turning it into something like `${str dstr}` "for optimization".

Reply:
You mean e.g. `it :focus { expect(foo).to eq(42) }`?

Reply:
But if method invocations are allowed (e.g. `it(description)`) we’d probably need `${str dstr send}`.

Reply:
No, I mean `_` is good, but just in case it's good to have a spec that fails if this gets changed to `${str dstr send}` and symbol would still be considered a valid description if passed as a first argument.

Side note: I remember there's also an `xstr`, but can't remember what it stands for.

Reply:
`xstr` is for when a string is enclosed in `